src/main/java/com/google/codeu/servlets/StatsPageServlet.java

```diff
@@ -0,0 +1,40 @@
+package com.google.codeu.servlets;
+
+
+import java.io.IOException;
+
+
+import javax.servlet.annotation.WebServlet;
```

Comment:
Please delete the `WebServlet` import and remove the `@WebServlet` annotation below.

Reply:
The changes have been made.